composer.json

```diff
@@ -29,16 +29,24 @@
         "php": ">=7.1.0",
         "ext-bcmath": "*",
         "ext-gmp": "*",
-        "guzzlehttp/guzzle": "~6.0",
         "psr/container": "^1.0",
-        "kornrunner/keccak": "^1.0"
+        "kornrunner/keccak": "^1.0",
+        "psr/http-message": "^1.0",
```

Comment:
you don't need it due php-http/httplug has already a dependency on psr/http-message

Reply:
I followed the implementation guide at http://docs.php-http.org/en/latest/httplug/library-developers.html - if we don't need id, it can go.

Reply:
The documentation can be outdated. If you take a look in the composer.json of http-plug you will see, that the dependency is already defined in it.

Reply:
Ok, adjusted.